Pilot_Plant_Dataset_Doc.ipynb

```diff
@@ -0,0 +1,2231 @@
+{
```

Comment:
<p>Part 1. Description of <strong><em>Del Mar College Glycol Pilot Plant </em></strong>Dataset Document</p><p> </p><p>Part 2. The <strong><em>Del Mar College Glycol Pilot Plant </em></strong>Challenges</p><p> </p><p>What Del Mar College teaches <strong><em>future</em></strong> operators with the pilot plant.</p><p> </p><p>Part 5.</p><p> </p><p><strong><em>Academic</em></strong> Hub Datasets are organized around a set of assets, <strong><em>with</em></strong> each of the <strong><em>assets</em></strong> having at least <strong><em>one</em></strong> default data view. A data view is an OCS feature to extract interpolated data from multiple related streams, in a tabular form ready for data-driven applications or any data analysis tool. Specialized data views (subsets of the default data views) can also be defined, <strong><em>as is</em></strong> the case for this dataset.   </p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/academic-hub/datasets/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
done